.github/workflows/test.yml

```diff
@@ -54,34 +54,32 @@ jobs:
           git diff --compact-summary --exit-code || \
             (echo; echo "Unexpected difference in directories after code generation. Run 'go generate ./...' command and commit."; exit 1)
 
-  # Run acceptance tests in a matrix with Terraform CLI versions
   test:
     name: Terraform Provider Acceptance Tests
     needs: build
     runs-on: ubuntu-latest
     timeout-minutes: 15
-    strategy:
-      fail-fast: false
-      matrix:
-        # list whatever Terraform versions here you would like to support
-        terraform:
-          - "1.0.*"
-          - "1.1.*"
-          - "1.2.*"
-          - "1.3.*"
-          - "1.4.*"
+    # enable strategy when run acceptance tests
+    # strategy:
+    #   fail-fast: false
+    #   matrix:
+    #     # list whatever Terraform versions here you would like to support
+    #     terraform:
+    #       - "1.0.*"
+    #       - "1.1.*"
+    #       - "1.2.*"
+    #       - "1.3.*"
+    #       - "1.4.*"
```

Comment:
the environment for the Acceptance Test is not ready, the Acceptance Test will not be run in CI until it is prepared